client/src/components/NavBar/useStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  toolbar: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    minHeight: 56,
+    backgroundColor: '#FFFFFF',
+  },
+  loginsignup: {
```

Comment:
```suggestion
  loginSignup: {
```
Nitpick: Please make sure you use camelCase when writing CSS in JS 😄